src/utils/backendAuthMixin.js

```diff
@@ -0,0 +1,47 @@
+import { mapState } from 'vuex';
+import Backend from './backend';
+import { createDeepLinkUrl } from './util';
+import { client } from './aeternity';
+
+export default (ignoreCallInQuery) => ({
+  computed: mapState(['useSdkWallet', 'account']),
+  ...!ignoreCallInQuery && {
+    async beforeRouteEnter(to, from, next) {
+      const {
+        method, address, challenge, signature,
```

Comment:
Address can't be taken from the store because `this` is not ready at this moment